src/iexamine.cpp

```diff
@@ -573,6 +573,8 @@ void iexamine::toilet(player &p, const tripoint &examp)
 
     if( water == items.end() ) {
         add_msg(m_info, _("This toilet is empty."));
+    } else if( water->item_tags.count( "FROZEN" ) ) {
```

Comment:
This looks inconsistent. In the code above you used `get_current_phase() == LIQUID`, here you check some item tag. Why the difference? Is there a difference? Why is this not just checking `get_current_phase() != LIQUID`?

Reply:
Not really any good reason.  I was actually really sad to see how toilet water works and just went with this.  Should probably be a check for SOLID instead, but I think toilets/sinks need to be looked at next in general to be less weird.